typings/index.d.ts

```diff
@@ -1755,6 +1764,41 @@ declare module 'discord.js' {
     public readonly createdAt: Date;
   }
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: unknown);
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public available: ?boolean;
+    public description: string;
+    public format: StickerFormatType;
+    public readonly guild: ?Guild;
+    public guildID: ?Snowflake;
```

Comment:
All nullable properties should be `actualType | null`

Reply:
```suggestion
    public guildID: Snowflake | null;
```